content/en/blog/_posts/2023-03-10-image-registry-change.md

```diff
@@ -0,0 +1,190 @@
+  need to be pointed directly at registry.k8s.io.
+- The redirect is a stopgap to assist users in making the switch. The deprecated k8s.gcr.io registry
+  will be phased out at some point. **Please update your manifests as soon as possible to point to
+  registry.k8s.io**.
+- If you host your own image registry, you can copy images you need there as well to reduce traffic
+  to community owned registries.
+
+If you think you may be impacted, or would like to know more about this change, please keep reading.
+
+## Why did Kubernetes change to a different image registry?
+
+k8s.gcr.io is hosted on a custom [Google Container Registry
+(GCR)](https://cloud.google.com/container-registry) domain that was set up solely for the Kubernetes
+project. This has worked well since the inception of the project, and we thank Google for providing
+these resources, but today, there are other cloud providers and vendors that would like to host
+images to provide a better experience for the people on their platforms. In addition to Google’s
+[renewed commitment to donate $3
+million](https://www.cncf.io/google-cloud-recommits-3m-to-kubernetes/) to support the project's
+infrastructure last year, Amazon Web Services announced a matching donation [during their Kubecon NA
+2022 keynote in Detroit](https://youtu.be/PPdimejomWo?t=236). This will provide a better experience
+for users (closer servers = faster downloads) and will reduce the egress bandwidth and costs from
+GCR at the same time.
+
+For more details on this change, check out [registry.k8s.io: faster, cheaper and Generally Available
+(GA)](/blog/2022/11/28/registry-k8s-io-faster-cheaper-ga/).
+
+## Why is a redirect being put in place?
+
+The project switched to [registry.k8s.io last year with the 1.25
+release](https://kubernetes.io/blog/2022/11/28/registry-k8s-io-faster-cheaper-ga/); however, most of
+the image pull traffic is still directed at the old endpoint k8s.gcr.io. This has not been
+sustainable for us as a project, as it is not utilizing the resources that have been donated to the
+project from other providers, and we are in the danger of running out of funds due to the cost of
+serving this traffic.
+
+A redirect will enable the project to take advantage of these new resources, significantly reducing
+our egress bandwidth costs. We only expect this change to impact a small subset of users running in
+restricted environments or using very old clients that do not respect redirects properly.
+
+## What images will be impacted?
+
+**ALL** images on k8s.gcr.io will be impacted by this change. k8s.gcr.io hosts many images beyond
+Kubernetes releases. A large number of Kubernetes subprojects host their images there as well. Some
+examples include the `dns/k8s-dns-node-cache`, `ingress-nginx/controller`, and
+`node-problem-detector/node-problem-detector` images.
+
+## What will happen to k8s.gcr.io?
+
+Separate from the the redirect, k8s.gcr.io will be frozen [and will not be updated with new images
```

Comment:
I don't know how much we want to go into here, but actually for anyone hitting the redirect to registry.k8s.io instead of the existing backing GCRs, all new images will be available at k8s.gcr.io by way of the redirect.

Somewhat confusing situation

Reply:
@BenTheElder i wanted to leave it vague... as they just need to know that they need to get off!